boards/xtensa/intel_s1000_crb/dts_fixup.h

```diff
@@ -26,4 +26,6 @@
 
 #define CONFIG_SPI_NOR_SPI_SLAVE		\
 	DT_SNPS_DESIGNWARE_SPI_E000_SPI_FLASH_0_BASE_ADDRESS
+
+#define DT_FLASH_DEV_NAME DT_SNPS_DESIGNWARE_SPI_E000_SPI_FLASH_0_LABEL
```

Comment:
This needs to be in a different commit. Where is this being used? I do not see this being used in this PR

Reply:
ok.